faq.md

```diff
@@ -859,7 +879,7 @@ pub fn f() {
 
 In the first example, the module is defined in the same file it's used. In the second example, the module declaration in the main file tells the compiler to look for either `hello.rs` or `hello/mod.rs`, and to load that file.
 
-A `use`ing declaration just tells the compiler to bring everything from a particular module into the current module. Without a `mod` declaration first, the compiler doesn't know if the `use`d module exists, and so can't import its contents into the current module.
+A `use` declaration just tells the compiler to bring *existing* names (reachable through some path) into scope. A `mod` declaration tells the compiler to *define* a module, and also brings that module's name into scope the same as `use` would.
```

Comment:
The last sentence-fragment feels odd. I don't think we would say it "also brings that item's name into scope the same as `use` would" for any other kind of declaration, like `struct`s or `fn`s (or even `mod`s defined inline). That's just what a declaration *does*. I would maybe characterize `mod foo;` as being a forward declaration, with the definition/body being provided in a separate file.